modules/representation_management/app/models/representation_management/accredited_entity_query.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  class AccreditedEntityQuery
+    include ActiveModel::Model
+
+    MAXIMUM_RESULT_COUNT = 10
+    WORD_SIMILARITY_THRESHOLD = 0.7
+
+    def initialize(query_string)
+      @query_string = query_string
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def results
+      return [] if @query_string.blank?
+
+      sql = <<-SQL.squish
```

Comment:
The query could be moved to a private method. That would get rid of the cop disable

Reply:
I tried moving it to a private method but the Metrics/MethodLength cop still didn't like it.  I can squish it down to 
```ruby
def sql_query
      <<-SQL.squish
      WITH combined AS (
        SELECT id, full_name AS name, 'AccreditedIndividual' AS model_type, levenshtein(full_name, ?) AS distance
        FROM accredited_individuals
        WHERE word_similarity(?, full_name) >= ? AND location IS NOT NULL
        UNION ALL
        SELECT id, name AS name, 'AccreditedOrganization' AS model_type, levenshtein(name, ?) AS distance
        FROM accredited_organizations
        WHERE word_similarity(?, name) >= ? AND location IS NOT NULL
      )
      SELECT id, model_type, distance
      FROM combined
      ORDER BY distance ASC
      LIMIT ?;
      SQL
    end
```
but that looks a lot less understandable.  I think disabling the cop is preferrable for how it makes the code easier to read and understand.  What do you think @jvcAdHoc ?

Reply:
I'd agree with readability over rubocop here. I like what you have now with the `sql_query` private method so that way the disabling is limited just to the query.